src/plugins/data_views/common/data_views/data_views.ts

```diff
@@ -720,7 +723,7 @@ export class DataViewsService {
     id: string,
     title: string,
     options: GetFieldsOptions,
-    fieldAttrs: FieldAttrs = {},
+    fieldAttrs: Record<string, FieldAttrSet> = {},
```

Comment:
```suggestion
    fieldAttrs: FieldAttrsAsObject = {},
```
Could this be `FieldAttrsAsObject` too?